### DISCUSSION TITLE
Doesnt work Update: turns out because in fullscreen no button to apply

### DISCUSSION THREAD

**Discussion by kangarooo** [Sophia-Community/SophiApp#15] [General]
🏷️ Bug 
Im opening, setting, closing. Doesnt work.
Update: turns out in fullscreen mode isnt buttons comming up apply.
Show buttons all the time, just greyed out.
In fullscreen make them seenable.
When in window mode, then only can see buttons.
When i restore, then can see buttons again.
To restore i need to hold shift+right menu click to open restore option. Theres no restore and maximize buttons on top.
I maximized allways by pulling window to top.
![image](https://user-images.githubusercontent.com/73187/146671298-d83d355d-b9e0-4c8c-8a91-e137789d113e.png)
![image](https://user-images.githubusercontent.com/73187/146671253-04b7cfc4-ba1b-4e5d-b8db-f862d597afeb.png)

**Comment by farag2** [comment]
Attach a log file.

**Comment by farag2** [comment]
Do you see an UAC prompt, where you need to type admin credentials, when you launch the app?

**Reply by kangarooo** [reply]
Yes. Asks for Admin password, but im opening in User10 where is no password. Cant choose different user. When i enter Admin password, app works, but nothing changes. Then i needed to go to Admin for other things and i saw that all i made was applied to Admin.

**Reply by farag2** [reply]
If it's so, the app works as it was designed. The only way to configure the current user is to grant the admin rights to him. This is how Windows works. Otherwise you will configure the admin account. Hopefully, I've explained.:)

**Reply by kangarooo** [reply]
No. It made changes from User10 in admin since it asked for admin password and i had it and i entered it and i rocked up admin account whitch i cant change. Now im fired from work, since app made changes in admin.

**Comment by farag2** [comment]
Anyway, it's strange for me why you were able to launch it if you're not an admin. I need to test it again.

**Reply by kangarooo** [reply]
It asked for admin password, so i cant run it without password.
I asked admin password.
Admin gave me password.
Now i entered in User2 and User10 opening this app the Admin password that it asked and nothing changed in User2 or User10. Only SophiaApp became dark choosing dark mode, but no other apps.
Then i checked admin account- all i did was made in Admin account. I didnt wanted to do anyhing in admin account.
Now im fired from work.

**Comment by farag2** [comment]
Also, check out the 80 build. It's still buggy, but removing UWP works perfectly.

**Reply by kangarooo** [reply]
80 and 81 still asks admin password
Puttin User2 as admin doesnt change anyhing, asks for Admin password.
Making password for User2 still asks for Admin password
Making User2 with password and admin still asks Admin password.
Each time testing logged out and in before testing.
UAC you mean? I have to remove User Account Control? Whats UWP? I hav eto removeit?

**Reply by farag2** [reply]
It will always require elevation. This is how Windows works. All you need is to make the current user an admin.

**Reply by kangarooo** [reply]
Now i tried Sophia Script, did updates of bloatware, so it would allow it to run, then asked for admin password and still said that this user also needs to be Admin.
Will check in 5 min again.
I hope this `Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process -Force
` means that it will be only for this User2
![image](https://user-images.githubusercontent.com/73187/146964223-79936bcb-7f07-4eb0-9932-3d8ee858a5d4.png)

**Reply by kangarooo** [reply]
Well sophia script did all only in this user when i made it admin also. So works at least sophia script. I would like only to have classic photo viewer. One bad thing- it removed touchpad scoll function working even if disabling and enabling scroll function again. Ill write it there also.
Ill try now sophiapp on user10

**Reply by farag2** [reply]
Good. :) But I don't touch any touch-driven features. Where can it be enabled back in Settings?

**Reply by farag2** [reply]
> I would like only to have classic photo viewer. 

This app is dead and the current Photo muxh better in all cases.

**Reply by kangarooo** [reply]
> > I would like only to have classic photo viewer.
> 
> This app is dead and the current Photo muxh better in all cases.

Main app was removed. Snip and sketch i also removed. I like classic.


> Good. :) But I don't touch any touch-driven features. Where can it be enabled back in Settings?

It didnt work after sophia script restart. It has own Dell touchpad app. Disabling scroll apply and enabling apply didnt work. Now did a test of SophiApp on User10 and logged back in User2 that had Sophia script, now User2 touchpad works and disabling and enabling now is back on each save without logging out.
User10 sophiApp 0.81 worked on User10 now only. I dont understand why it didnt before on User2, but made changes in Admin. On User2 testing stable 0.7 maybe saved some config that made 0.8 and 0.81 also continue to do changes in Admin?
Now ill remove windows and put on linux Kubuntu in 2 hours. Before that i can do some more test of script or app after 1h30m.
![image](https://user-images.githubusercontent.com/73187/146969856-2d0a1bb3-5d8a-405d-8b31-5ece2c190a05.png)

**Reply by kangarooo** [reply]
Update- Admin scroll not working. Disabling enabling didnt work. Log out in worked after disbling enabling.
Logged in User2 that had sochia scrip and didnt work scroll and got back after disabling enabling relogging, now that again not working scroll on touchpad.
Ill be back in 1h30m.

**Reply by kangarooo** [reply]
Update- Made User11 and made hardcore remove all apps for all users. All worked. Again was not working until restart touchpad scroll.
Seems script is ok. One drive and office removed for all. All is ok with Sophia scipt, but since no good scripts are perfect then im going to Kubuntu not to waste time with viruses, cancer and other this that widows has.

**Comment by farag2** [comment]
> I hope this Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope Process -Force means that it will be only for this User2

This command means that you changed the execution policy only for this process. I advise this due to secureness.

As I told you twice, your current user doesn't have an admin rights. Did you get the point? :) If the script requires a password to open a console in another user scope, it means that you will make any changes to that account, not the current one you are logged in.

Log into your admin account and change your current user type to admin. Only this way.

**Reply by kangarooo** [reply]
Ah, idea there needs to have a warning. Sophia Script was warning about that
https://github.com/Sophia-Community/SophiApp/discussions/15#discussioncomment-1853747
